### PR TITLE
Docs formatting fixes

### PR DESCRIPTION
**Pull request by DanailPenev**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Formatting lists for a living
Linted Glossary as a hobby

#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?
#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/projects/overview.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 The SpatialOS GDK for Unity is accompanied by the following projects to help you get started:
 
-* [FPS Starter project]({{urlRoot}}/projects/template-fps/overview)
```
Comment (jessicafalk) [inline docs/projects/overview.md]: that should probably stay at top for now as we don't have a blank project

File: docs/content/deploy.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ When you want to try out your game, you need to run a deployment of the game. Th
 
 Before you deploy your game, you need to build its workers; see documentation on [building workers]({{urlRoot}}/content/build.md).
```
Comment (oblm) [inline docs/content/deploy.md]: Please search and delete all superfluous instances of `.md` in improbadoc urls.

File: docs/projects/overview.md (hunk 1 above)
Comment (DanailPenev) [reply docs/projects/overview.md]: The reason for swapping them was that the quicklink at the bottom of the page was pointing to the blank project page. Will revert the swap.

File: docs/content/deploy.md (hunk 2 above)
Comment (samc-improbable) [reply docs/content/deploy.md]: Now fixed.

**Pull request merged**